### DISCUSSION TITLE
Official support for MongoDB 8

### DISCUSSION THREAD

**Discussion by alexandru-2016** [payloadcms/payload#9088] [Q&A] [answered]
Hello,

We are using Payload CMS connected to a Mongo DB hosted on Atlas Mongo Cloud, and we've been recently notified by the Atlas platform that they will be upgrading all clusters to MongoDB 8 this month.

As far as I can tell, Payload is using mongoose 6, which doesn't seem to support MongoDB 8, from the compatibility table [here](https://mongoosejs.com/docs/compatibility.html)

Are there any plans to officially support mongoose 8.7, as listed by the compatibility table?

Thanks!

**Comment by DanRibbens** [comment]
Thank you for bringing this to our attention! We're on it.

**Reply by DanRibbens** [reply, marked as answer]
Related PR https://github.com/payloadcms/payload/pull/9115
